rider/src/main/kotlin/com/jetbrains/rider/plugins/godot/run/GodotRunConfigurationGenerator.kt

```diff
@@ -44,7 +48,8 @@ class GodotRunConfigurationGenerator : LifetimedService() {
                 val godotDiscoverer = GodotProjectDiscoverer.getInstance(project)
                 godotDiscoverer.godotDescriptor.viewNotNull(lifetime) { lt, descriptor ->
                     logger.info("descriptor = $descriptor")
-                    val basePath = descriptor.mainProjectBasePath
+                    val tempRelPath = Paths.get(descriptor.mainProjectBasePath).relativeTo(project.solutionDirectory.toPath())
```

Comment:
Make sure this is not a throwing method. I remember some cases when `Path::relativize` was throwing exceptions in cases it cannot construct a relative path (i.e. when the paths are on unrelated trees, e.g. different drives on Windows). Not sure if `relativeTo` is same as `relativize` or not.